pkg/client/client.go

```diff
@@ -230,7 +232,7 @@ type Client interface {
 	ProjectCreate(ctx context.Context, name string) (*apiv1.Project, error)
 	ProjectDelete(ctx context.Context, name string) (*apiv1.Project, error)
 
-	Info(ctx context.Context) (*apiv1.Info, error)
+	Info(ctx context.Context) ([]apiv1.Info, error)
```

Comment:
Well, the problem is that in Hub server information can vary between projects. Right now, because of how we do upgrades, the server versions happen to be consistent, but we don't want to rely on that.